scripts/wheel/build_wheel_manylinux.sh

```diff
@@ -21,14 +21,38 @@ cp ../cmake/config.cmake .
 echo set\(RAF_USE_LLVM \"llvm-config-8\"\) >> config.cmake
 echo set\(HIDE_PRIVATE_SYMBOLS ON\) >> config.cmake
 echo set\(RAF_USE_GTEST ON\) >> config.cmake
-echo set\(RAF_USE_CUDA ON\) >> config.cmake
-echo set\(RAF_CUDA_ARCH 75\) >> config.cmake
-echo set\(RAF_USE_CUBLAS ON\) >> config.cmake
-echo set\(RAF_USE_CUDNN ON\) >> config.cmake
 echo set\(RAF_USE_SANITIZER OFF\) >> config.cmake
 echo set\(RAF_USE_MPI OFF\) >> config.cmake
 echo set\(RAF_USE_NCCL OFF\) >> config.cmake
-echo set\(RAF_USE_CUTLASS ON\) >> config.cmake
+
+# Set environment variables to specify the version and platform to build RAF
+# RAF_BUILD_VERSION. Option: [stable/nightly/dev]
+export RAF_BUILD_VERSION=
+
+# RAF_BUILD_PLATFORM. Option: [cpu/cu113]
+export RAF_BUILD_PLATFORM=
```

Comment:
This will remove user variables IIUC. Better to remove it.
You could move the comments down and make them along with the conditions.